Charts: update the 'Simple Array' demo description (#1854)
DevExpress/devextreme-demos: JSDemos/Demos/Charts/SimpleArray/description.md

> You can populate the Chart component with data locally or remotely. If you do not need to get data from the server or update data dynamically, bind the component to a local JavaScript array. To bind Chart to a local array, pass this array to the Chart's [dataSource](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource) property. 

Reviewer: First you give the reader two choices: local or remote data. Then you repeat the remote option (server), but then introduce a whole new concept (update dynamically), and then talk about local data (array) again. So in the first sentence, I had two options. IN the second, I also see a third - dynamic data... But I don't know if it relates to remote data, or local, or both. So I suggest that we combine all of that.

```suggestion
If you do not need to populate the chart with data from a remote server, and the chart doesn't need to update data frequently, then simply bind the component to a local JavaScript array. Pass the array to the Chart's [dataSource](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource) property. 
```

Author: Can you please explain to me when we start the component's name with the capital letter, and when we use only lowercase letters? I thought that when we are talking about the component, then we use the capital letter, e.g. the Chart component. And when we talk about the chart as a visual element in general, not necessarily the DX component, we use only the lowercase.
I thought that I'm writing here about our component specifically, because it's not some generic chart, but the Chart that can be populated with data in specific ways. And later, when I'm talking about the dataSource, I also write "the Chart's dataSource", since the dataSource property belongs to the component. 
I just think it's not consistent to write about the component "Chart/chart" in one rather small text.

Reviewer: "Chart" is fine. Sorry for the confusion :-) I didn't really mean to change that. 

In general, I'd say it's fine to use lowercase "chart" as a synonym, if the reader knows by now what kind of chart we're talking about. That could help avoid repetition if, say, your control's name is ASPxChartControl. That way you can alternate that with a simple word "chart", and make text a bit easier to read. 

But if your control's name is already simple enough ("Chart"), it doesn't really make sense to use a lowercase "synonym" :-) You're right.  

```suggestion
If you do not need to populate the Chart with data from a remote server, and the data doesn't change frequently, then simply bind the component to a local JavaScript array. Pass the array to the Chart's [dataSource](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource) property. 
```

> Once you assign the data source, specify the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) type and its nested options: [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#argumentField) and [valueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#valueField). If you specify these properties, the component can determine the object fields that indicate Chart arguments and values in the array. The default values for these properties are `arg` and `val` respectively. You can use these data field names to structure your array as shown in this demo.

Reviewer 2: ```suggestion
Once you assign the data source, specify the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) type and its nested options: [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#argumentField) and [valueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#valueField). If you specify these properties, the component can determine the object fields that indicate Chart arguments and values in the array. The default values for these properties are `arg` and `val`, respectively. You can use these data field names to structure your array as shown in this demo.
```

> Then, specify the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) type. Each [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) type has the [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#argumentField) and the [valueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#valueField) properties. If you specify these properties, the component can determine which object fields in the array indicate Chart arguments and values. The default values for these properties are `arg` and `val` respectively. You can use these data field names to structure your array as shown in this demo.

Reviewer: I don't like the sentences that simply start with "then". A simple "then" is implied if you describe a sequence of actions. You can be specific, or simply omit "then".

I also tried to simplify / shorten the link between first and second sentences .

```suggestion
Once you assign the data source, specify the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) type and its nested options:  [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#argumentField) and [valueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#valueField). If you specify these properties, the component can determine which object fields in the array indicate Chart arguments and values. The default values for these properties are `arg` and `val` respectively. You can use these data field names to structure your array as shown in this demo.
```
